.github/bin/build-matrix-from-impacted.py

```diff
@@ -61,13 +61,16 @@ def build(matrix_file, impacted_file, output_file):
         if impacted and not any(module in impacted for module in modules):
             logging.info("Excluding matrix section: %s", item)
             continue
-        include.append(
-            {
-                # concatenate because matrix values should be primitives
-                "modules": ",".join(modules),
-                "profile": item.get("profile", ""),
-            }
-        )
+
+        obj = {
+            # concatenate because matrix values should be primitives
+            "modules": ",".join(modules),
+            "profile": item.get("profile", ""),
+        }
+        i = item.get("i", None)
+        if i is not None:
+            obj["i"] = i
+        include.append(obj)
```

Comment:
Let's do this separately. Will need more work to allow arbitrary keys instead of hardcoded ones.

Reply:
I'll open up another pr

Reply:
https://github.com/trinodb/trino/pull/11446